app/src/main/res/layout/item_status.xml

```diff
@@ -318,14 +322,12 @@
 
         <com.varunest.sparkbutton.SparkButton
             android:id="@+id/status_reblog"
-            android:layout_width="40dp"
```

Comment:
I thought you would keep button size the same and only make icon inside it smaller. It is okay for me, just worrying for people who claim themselves clumsy.